src/input.service.ts

```diff
@@ -376,6 +376,7 @@ export class InputService {
   public bootstrap(root: HTMLElement = document.body) {
     if (typeof navigator.getGamepads === 'function') {
       // Poll connected gamepads and use that for input if possible.
+      (<any>navigator).gamepadInputEmulation = 'gamepad';
```

Comment:
Should probably do a platform check before, not sure if some browser will error if you simply write something to the navigator.